state.css

```diff
@@ -0,0 +1,5 @@
+@media only screen and (min-width: 640px) {
```

Comment:
Per the instructions:

```
The nav menu should be hidden by default in mobile mode and should display when the user clicks the hamburger icon.
```
You should add some styling to your `.nav` so that it shows as normal when the screen exceeds 640 px.